src/Language/Haskell/Liquid/Types/Errors.hs

```diff
@@ -990,6 +990,10 @@ ppError' _ dCtx (ErrSaved _ name s)
         $+$ dCtx
         $+$ {- nest 4 -} s
 
+-- XXX(matt.walker): If this was because of the limitations taken care of with
+-- the addition of `processPragmas`, then we should remove this. It could be a
+-- "it's too late after the plugin is running to change these values" thing
+-- though, I'm not sure.
```

Comment:
RFC

Reply:
cc @nikivazou

Reply:
If you are referring to the fix in https://github.com/ucsd-progsys/liquidhaskell/pull/1984/commits/ecbc2fda203b9b034cb72fd6e60ec1509917e453, it looks to me like this should be unrelated. I would remove this comment.